lib/array.js

```diff
@@ -38,6 +38,60 @@ const map = (
   }
 };
 
+const NS_PER_SEC = 1e7;
+
+const asyncMap = (
+  // Non blocking map
+  items, // array, incoming
+  fn, // function, (value, index) => {}
+  {
+    min = 5,
+    max = 30,
+  } = {}
+) => {
```

Comment:
You don't seem to return the resulting array from the function, at the moment it works as the `forEach()` function.